src/org/scada_lts/web/mvc/controller/ScriptController.java

```diff
@@ -59,6 +60,7 @@
 		// TODO ta metoda powinna przyjmowac jako argument tresc skryptu z
 		// odpowiednimi argumentami nie powinno to isc przez baze danych !!!.
 		User user = Common.getUser(request);
+		Permissions.ensureAdmin(user);
```

Comment:
The same issue is here. Are we sure that we want to disable scriptExecution invoked by non-admin-user in ScriptController? I know that we do not have "script objects permission" but what if someone who is not an admin wants to execute existing script? That is also a business logic change.

Reply:
You may be right, I'll check tomorrow.